tests/test_cache.py

```diff
@@ -11,16 +16,28 @@
 def r(request):
     cache = request.param.get("cache")
     kwargs = request.param.get("kwargs", {})
+    protocol = request.param.get("protocol", 3)
+    single_connection_client = request.param.get("single_connection_client", False)
     with _get_client(
-        redis.Redis, request, protocol=3, client_cache=cache, **kwargs
+        redis.Redis,
+        request,
+        single_connection_client=single_connection_client,
+        protocol=protocol,
+        client_cache=cache,
+        **kwargs,
     ) as client:
         yield client, cache
         # client.flushdb()
 
 
+@pytest.fixture()
+def local_cache():
+    return _LocalCache()
+
+
 @pytest.mark.skipif(HIREDIS_AVAILABLE, reason="PythonParser only")
+@pytest.mark.onlynoncluster
```

Comment:
Why? I think we want some of the tests in this class to also run in the cluster

Reply:
Good point. I'll revert this.

Reply:
Reverted.